auth-tokens/src/main/java/com/palantir/tokens/auth/BearerToken.java

```diff
@@ -39,6 +41,12 @@
     @JsonValue
     public abstract String getToken();
 
+    @Value.Derived
+    @Value.Auxiliary
```

Comment:
Given that we already have our own `equals` `hashCode` and `toString` - this `@Value.Auxiliary` doesn't actually do anything right? https://immutables.github.io/immutable.html#auxiliary-attributes